frontend/src/metabase/visualizations/lib/settings/graph.ts

```diff
@@ -157,7 +172,7 @@ export const GRAPH_DATA_SETTINGS = {
         settings["graph.max_categories"] !== 0
           ? settings["graph.max_categories"]
           : Infinity;
-      const onOtherColorChange = color =>
+      const onOtherColorChange = (color: CSSProperties["color"]) =>
```

Comment:
👍 for using the proper type. I'd have just slapped `string` and called it a day 😆 But using a proper is a great addition 😉